app/mm-redux/actions/users.ts

```diff
@@ -1378,6 +1378,12 @@ export function enableUserAccessToken(tokenId: string): ActionFunc {
     };
 }
 
+export function getKnownUsers(): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.getKnownUsers,
+    });
+}
+
```

Comment:
explain? and the binding is not really doing anything, why not just call `Client4.getKownUsers`?

Reply:
I was following the pattern but yes, probably is not needed at all the action itself.

Reply:
Removed